client/allocrunner/network_hook_linux_test.go

```diff
@@ -146,18 +152,19 @@ func TestNetworkHook_Prerun_Postrun_ExistingNetNS(t *testing.T) {
 				must.EqError(t, err, tc.expectPrerunError)
 			}
 
-			test.Eq(t, tc.expectPrerunDestroyNetworkCalls,
-				callCounts.Get()["DestroyNetwork"], test.Sprint("DestroyNetwork calls after prerun"))
-			test.Eq(t, tc.expectPrerunCreateNetworkCalls,
-				callCounts.Get()["CreateNetwork"], test.Sprint("CreateNetwork calls after prerun"))
+			must.Eq(t, tc.expectPrerunDestroyNetworkCalls,
+				callCounts.Get()["DestroyNetwork"], must.Sprint("DestroyNetwork calls after prerun"))
```

Comment:
why change these from `test` to `must`?

sometimes I like seeing all the different things that failed at once, rather than fix one problem at a time and re-run the test after each one.